mungegithub/submit-queue/deployment.yaml

```diff
@@ -14,13 +14,34 @@ spec:
         command:
         - /mungegithub
         - --token-file=/etc/secret-volume/token
+        - --organization=kubernetes
+        - --project=kubernetes
+        - --state=open
         - --pr-mungers=blunderbuss,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,ok-to-test,rebuild-request,path-label,size,stale-pending-ci,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue,issue-cacher,flake-manager,old-test-getter,close-stale-pr
         - --dry-run=true
-        - --weak-stable-jobs=kubernetes-kubemark-500-gce
         - --required-contexts="Jenkins GCE Node e2e"
-        - --number-of-old-test-results=5
         - --http-cache-dir=/cache/httpcache
+        - --path-label-config=path-label.txt
+        - --block-path-config=block-path.yaml
+        - --path-label-config=path-label.txt
+        - --kubernetes-dir=/gitrepos/kubernetes
         - --test-owners-csv=/gitrepos/kubernetes/test/test_owners.csv
+        - --number-of-old-test-results=5
+        - --fixes-issue-reassign=false
+        - --blunderbuss-reassign=false
```

Comment:
I was trying to bring out and make explicit all the command-line options that we might need to change as we run on other repositories. I think relying on defaults in code could make it a little difficult to understand behavior later.